nri-config-generator/integration-tests/testdata/config_with_port.yml

```diff
@@ -1,7 +0,0 @@
-parent:
-  child:
-    - element1: a
-    - element2: b
-ravendb_url: "http://live-test.ravendb.net"
```

Comment:
would make sense to rename the new file to `config_with_scrape_timeout_and_port`?

Reply:
I don't think so. The new file is not replacing this one, I removed it because it was not being referenced in any test.